doc/sphinx/source/graph.rst

```diff
@@ -93,11 +93,11 @@ These all assume that an object of :class:`.GraphDataScience` is available as `g
 
     Streams the given node property.
 
-.. py:function:: gds.graph.project(graph_name: str, node_spec: Any, relationship_spec: Any, **config: Any) -> Tuple[Graph, Series[Any]]
+.. py:function:: gds.graph.project(graph_name: str, node_spec: Any, relationship_spec: Any, **config: Any) -> GraphResult
 
     Creates a named graph in the catalog for use by algorithms.
-
-.. py:function:: gds.graph.project.cypher(graph_name: str, node_spec: Any, relationship_spec: Any, **config: Any) -> Tuple[Graph, Series[Any]]
+# FIXME - how to document GraphResult here
```

Comment:
How can I document this dataclass correctly. I did not know where it fit. A seperate page seemed overkill

Reply:
Separate page is the structure we have for it now. You can compare to `NodePropertyStep` or `LinkFeature` classes. It's been fed back that it's not great but no solution has been made yet. I would pile it up next to those and solve them all later.